Mention required nuget.org package source

donnie-msft opened the pull request:
If nuget.org is not an enabled package source on the machine, an error will occur while trying to create the project. 
NuGet does not currently support referencing a package from a VSIX, so the remote source is still necessary: https://github.com/NuGet/Home/issues/4693

donnie-msft committed:
Mention required nuget.org package source

If nuget.org is not an enabled package source on the machine, an error will occur while trying to create the project. 
NuGet does not currently support referencing a package from a VSIX, so the remote source is still necessary: https://github.com/NuGet/Home/issues/4693

PRMerger12 commented:
@donnie-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mcleanbyron committed:
Edited pull request

Moved nuget.org package source info to a separate installation step for consistency

PRMerger15 commented:
@mcleanbyron : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mcleanbyron committed:
Revised link format

PRMerger16 commented:
@mcleanbyron : Thanks for your contribution! The author(s) have been notified to review your proposed change.
